diff --git a/command_line_assistant/commands/chat.py b/command_line_assistant/commands/chat.py
index ab15454..818fa61 100644
--- a/command_line_assistant/commands/chat.py
+++ b/command_line_assistant/commands/chat.py
@@ -296,7 +296,11 @@ class BaseChatOperation(BaseOperation):
             response(str): The message to be displayed
         """
         self.legal_renderer.render(LEGAL_NOTICE)
+        self.text_renderer.render("─" * 72)
+        print("")
         self.markdown_renderer.render(response)
+        print("")
+        self.text_renderer.render("─" * 72)
         self.notice_renderer.render(ALWAYS_LEGAL_MESSAGE)
 
     @timing.timeit
